README.md

```diff
@@ -55,3 +55,13 @@ docker build -t jupyter/demo .
 docker build -t jupyter/minimal common/
 ```
 
+#### `jupyter/minimal`
+You will need to run this code snippet before building the docker image
+```
+git clone https://github.com/ericdill/scikit-xray-examples
```

Comment:
Take a look at line 38 in Dockerfile. We can do the cloning as part of the docker build process -- no need for manual setup.